l2_deploy.py

```diff
@@ -186,7 +186,7 @@ def _generate_l2_genesis(config: Config):
         print("Generating L2 genesis and rollup configs")
         try:
             log_file = f"{config.logs_dir}/generate_l2_genesis.log"
-            l2_allocs_path = os.path.join(config.artifacts_dir, 'allocs-l2.json')
+            l2_allocs_path = os.path.join(config.artifacts_dir, 'allocs-l2-granite.json')
```

Comment:
We should add two config variables here:
- target hard fork (with default value "granite")
- allocs file name (with default value derived from the target hard fork)